sdk/cosmos/azure-cosmos/azure/cosmos/partition_key.py

```diff
@@ -19,9 +19,11 @@
 # OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN THE
 # SOFTWARE.
 
+"""Create partition keys in the Azure Cosmos DB SQL API service.
+"""
 
 class NonePartitionKeyValue(object):
-    """Represents none value for partitionKey when it's missing in a containers.
+    """Represents None value for partitionKey when it's missing in a containers.
```

Comment:
Either "container" singular, or "in containers"

Reply:
fixed in https://github.com/Azure/azure-sdk-for-python/pull/8607/commits/c14bb0d8adcb5e63ac1d6a2fab3819ccca50707e